utils/models.py

```diff
@@ -118,6 +121,8 @@ def serialize(self):
         return {
             "key": self._meta.model_name.upper(),
             "children": [
-                self._resolve_field(self, field) for field in self.serialize_fields
+                self._resolve_field(self, field)
+                for field in self.serialize_fields
+                if self._resolve_field(self, field) is not None
```

Comment:
just curious: why exactly `is not None`? why not just `if self._resolve_field(self, field)`?

Reply:
That's an excellent question 🤦‍♂  I guess we can manage without that 👌